guardhelper.lua

```diff
@@ -277,14 +277,18 @@ end
 
 
 function guardhelper:highlight_state()
-  if guardhelper.show_suggested_target then
-    self:render_shields()
-  end
-  if guardhelper.show_most_wounded then
-    self:render_wounded()
-  end
-  if guardhelper.show_most_attacked then
-    self:render_attacked()
+
+
+  if table.size(ateam.team) > 1 then   
```

Comment:
moglbys zerknac czy w avatar object.data jest Team TRUE/FALSE?, wolalbym to tak sprawdzac...

Reply:
ateam.objs[ateam.my_id]["team"] - chyba to ale nie mam pod reka mudleta zeby sprawdzic